script/script.go

```diff
@@ -714,3 +757,10 @@ func getPathParam(f filters.FilterContext) func(*lua.LState) int {
 		return 0
 	}
 }
+
+func unsupported(message string) func(*lua.LState) int {
```

Comment:
total NIT picking: this name is more restrictive than what the function does. It could be just luaError, or maybe the message could be prefixed with "unsupported: "